gimpopenvino/tools/stable-diffusion-ov-server.py

```diff
@@ -232,78 +260,45 @@ def run(model_name,device_name):
 
                         prompt = data_output["prompt"]
-                        scheduler = data_output["scheduler"]
+                        #scheduler = data_output["scheduler"]
                         negative_prompt = data_output["negative_prompt"]
                         init_image = data_output["initial_image"]
+                        num_images = data_output["num_images"]
                         num_infer_steps = data_output["num_infer_steps"]
                         guidance_scale = data_output["guidance_scale"]
                         strength = data_output["strength"]
                         seed = data_output["seed"]
                         create_gif = False #data_output["create_gif"]
 
-                        if scheduler == "LMSDiscreteScheduler":
-                             #log.info('LMSDiscreteScheduler...')
-                             scheduler = LMSDiscreteScheduler(
-                                    beta_start=0.00085,
-                                    beta_end=0.012,
-                                    beta_schedule="scaled_linear",
 
-                                )
-                        elif scheduler == "PNDMScheduler":
-                            #log.info('PNDMScheduler...')
-                            scheduler = PNDMScheduler(
-
-                                beta_start=0.00085,
-                                beta_end=0.012,
-                                beta_schedule="scaled_linear",
-                                skip_prk_steps = True,
-
-                            )
-
-                        elif scheduler == "LCMScheduler":
-                            #log.info('LCMScheduler...')
-                            scheduler = LCMScheduler(
-                                beta_start=0.00085,
-                                beta_end=0.012,
-                                beta_schedule="scaled_linear"
-                            )
-
-                        elif scheduler == "UniPCMultistepScheduler":
-                            #log.info('UniPCMultistepScheduler')
-                            scheduler = UniPCMultistepScheduler(
-                                beta_start=0.00085,
-                                beta_end=0.012,
-                                beta_schedule="scaled_linear"
-                                )
-
-                        else:
-                             #log.info('EulerDiscreteScheduler...')
-                             scheduler = EulerDiscreteScheduler(
-                             beta_start=0.00085,
-                             beta_end=0.012,
-                             beta_schedule="scaled_linear"
-                             )
 
 
                         strength = 1.0 if init_image is None else strength
                         log.info('Starting inference...')
                         log.info('Prompt: %s',prompt)
                         log.info('negative_prompt: %s',negative_prompt)
```

Comment:
Let's gate this on not being the LCM model, since the LCM model doesn't have a negative prompt:
```
if model_name != "SD_1.5_square_lcm":
    log.info('negative_prompt: %s',negative_prompt)
```